lib/rpc.ts

```diff
@@ -129,18 +131,31 @@ export class Rpc extends EventEmitter implements IForwarderDest {
   }
 
   /**
-   * Until start() is called, received messages are queued. This gives you an opportunity to
-   * register implementations and add "message" listeners without the risk of missing messages,
+   * Until start() is called, received and sent messages are queued. This gives you an opportunity
+   * to register implementations and add "message" listeners without the risk of missing messages,
    * even if receiveMessage() has already started being called.
    */
   public start(sendMessage: SendMessageCB) {
-    this._sendMessage = sendMessage;
+    this._sendMessageCB = sendMessage;
     if (this._inactiveQueue) {
       for (const msg of this._inactiveQueue) {
         this._dispatch(msg);    // We need to be careful not to throw from here.
       }
       this._inactiveQueue = null;
     }
+    for (const msg of this._inactiveSendQueue) {
+      sendMessage(msg);
+    }
+    this._inactiveSendQueue = [];
+  }
+
+  /**
+   * Calling stop() resume the same state as before start was called: received and sent messages are
+   * queued.
+   */
+  public stop() {
+    this._inactiveQueue = [];
```

Comment:
Shouldn't _inactiveSendQueue be treated the same way?

Reply:
This line has changed. Now only `_sendMessageCB` is used for both sent and received messages to check the state.